WordPress/src/main/java/org/wordpress/android/ui/reader/SubfilterBottomSheetFragment.kt

```diff
@@ -46,6 +55,9 @@ class SubfilterBottomSheetFragment : BottomSheetDialogFragment() {
 
         viewModel = ViewModelProviders.of(requireActivity(), viewModelFactory).get(ReaderPostListViewModel::class.java)
 
+        performUpdate()
```

Comment:
We might want to consider invoking something like `viewModel.onSubFilterBottomSheetShown()` and moving both the lines there. It's technically a business logic. Wdyt?

Side question: Do we want to call it whenever the view is created (even after rotation?).

Reply:
Correct 😊, also agree with the side question, I don't think we should trigger the update on configuration changed or similar. Changed here feef4bceedb37674d9b5ef01706059c0837e6f9a